lib/tasks/test.rake

```diff
@@ -1,16 +1,29 @@
 namespace :test do
   # Use `> log/test.log` to empty test.log before re-running tests to get an accurate list
   # Note you must run tests with `COVERAGE=true` to generate scannable logs.
-  desc 'Scan test.log for rendered views'
-  task scan_log_for_render: :environment do
-    # match 'Rendered two_factor_authentication/otp_verification/show.html.erb'
-    # Rendered + space + word + [/ + non-whitespace](any number of times)
-    regex_finder = %r{Rendered\s\w*(/\S*)*}
+  desc 'Scan test.log for rendered views and show gaps in test coverage'
+  task scan_log_for_view_coverage: :environment do
```

Comment:
We can remove the `:environment` part (that loads all of Rails) and this is just some simple filesystem access, doesn't need the DB. I tested it locally and it worked

So this line would be:

```diff
-  task scan_log_for_view_coverage: :environment do
+  task :scan_log_for_view_coverage do
```

Reply:
Oh, I didn't know that's what `:environment` does.  Changing, thanks!